components/ArticleProfile.jsx

```diff
@@ -0,0 +1,45 @@
+import {useParams} from "react-router-dom"
+import {useState, useEffect, useContext} from "react"
+import { ArticlesContext } from "../contexts/ArticlesContext"
+import ArticleCard from "./ArticleCard"
+
+
+const UserProfile=()=>{
+    const {articles,setArticles}=useContext(ArticlesContext)
+
+    const {article_id}=useParams()
+    const [article, setArticle]=useState([])
+    const [isLoading, setIsLoading]=useState(true)
+ 
+    useEffect(()=>{
+        fetch(`https://xing-wei-app.onrender.com/api/articles/${article_id}`)
```

Comment:
I strongly recommend using axios over fetch - fetch will become more difficult to use as you get into queries etc. https://notes.northcoders.com/courses/js-front-end/axios